tests/runtest.cmd

```diff
@@ -191,6 +192,19 @@ if errorlevel 1 (
     exit /b 1
 )
 
```

Comment:
It seems like perf tests will also run as part of functional test runs. I don't think we want that